apps/taskmasters/flowlord/README.md

```diff
@@ -72,6 +73,16 @@ rule = "cron=0 0 1 * * *&Offset=-6"
 template = "?date={yyyy}-{mm}-{dd}T{hh}"
 ```
 
+### files 
+schedule a task after a specified file is written. This should be used with the filewatcher taskmaster or GCP file watching service. File matching is done using the filepath.Match which does not support `**` matching. Flowlord will attempt to pull the timestampe from the filepath which will can be used in the template with the time templates `{yyyy}|{dd}|{mm}|{hh}` also the matching file will be available through the metadata of the task `{meta:file}`
```

Comment:
Do you mean `which can be used in the template` not `which will can be used in the template`